dist/aem.js

```diff
@@ -519,6 +519,27 @@ async function fetchPlaceholders(prefix = 'default') {
   return window.placeholders[`${prefix}`];
 }
 
+/**
+ * Ensure all column div's within a block contain a block level element (p, h2, pre, etc.)
+ * Avoids situations where you end up with a column with just a text node but no wrapping p tag
+ * @param {Element} block the block element
+ */
+// eslint-disable-next-line import/prefer-default-export
+function addMissingParagraphs(block) {
+  block.querySelectorAll(':scope > div > div').forEach((blockColumn) => {
+    const hasContent = !!blockColumn.firstElementChild || !!block.textContent.trim();
```

Comment:
```suggestion
    const hasContent = div.hasChildNodes() && (!div.firstElementChild || !div.querySelector('picture'));
```

Reply:
I moved the picture check into the `hasWrapper` check rather than `hasContent` but otherwise this is (I think) mostly what you suggested here.